packages/airnode-deployer/docker/entrypoint.sh

```diff
@@ -4,4 +4,6 @@ DEPLOYER_BINARY="/usr/local/bin/airnode-deployer"
 USER_ID="${USER_ID:-0}"
 GROUP_ID="${GROUP_ID:-0}"
 
+test -f aws.env && export $(egrep -v '^\s*#' aws.env | xargs)
```

Comment:
(For the future): It's better to use the long argument versions, since it's totally not clear what `-v` does (compared to `--invert-match` which is self explanatory)